README.md

```diff
@@ -1,17 +1,44 @@
-## My Project
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px" alt="OpenSearch logo">
 
-TODO: Fill this README out!
+## OpenSearch API Specification
 
-Be sure to:
+- [Welcome!](#welcome)
+- [Project Resources](#project-resources)
+- [Code of Conduct](#code-of-conduct)
+- [Security](#security)
+- [License](#license)
+- [Copyright](#copyright)
 
-* Change the title in this README
-* Edit your repository description on GitHub
+## Welcome!
+
+`opensearch-api-specification` is an open source, community-driven collection of API model specifications for
+[OpenSearch](https://github.com/opensearch-project/OpenSearch) APIs. The API models are created using the [Smithy](https://awslabs.github.io/smithy/) Interface Definition Language (IDL). Smithy is an open source IDL which is protocol agnostic, extensible and supports code generation.
+
+To contribute to this project or to track the developments head over to [Projects](https://github.com/opensearch-project/opensearch-api-specification/projects)
+board. Follow the [Developer guide](DEVELOPER_GUIDE.md) and [Contributing guidelines](CONTRIBUTING.md) for instructions
+on building and contributing to opensearch-api-specification.
+
+## Project Resources
+
+* [Project Website](https://opensearch.org/)
+* [Downloads](https://opensearch.org/downloads.html)
+* [Documentation](https://opensearch.org/docs/)
```

Comment:
The Project Resources here redirect to the OpenSearch project resources and not the api-specification. Should we mention that?

Reply:
The link to resources would be same, we probably need to publish the artifacts to be available there